modules/apps/object/object-test/src/testFunctional/tests/ObjectUpgrade.testcase

```diff
@@ -13,7 +13,8 @@ definition {
 
 		AssertLocation(value1 = "${portalURL}/web/guest?SM_USER=test@liferay.com");
 
-		SearchAdministration.executeReindex();
+		//SearchAdministration.executeReindex();
+
```

Comment:
Why are we no longer reindexing here? We expect users to reindex after upgrades. Is it causing an issue with the update tests?

Reply:
Oops I commented that out while testing locally to save some time, and forgot to change it back.